pkg/controllers/machine_controller.go

```diff
@@ -154,12 +169,54 @@ func (r *MachineReconciler) Start(ctx context.Context) error {
 		}
 	}()
 
+	var wg sync.WaitGroup
+	wg.Add(1)
+	go func() {
+		sizeLogger := r.log.WithName("size")
```

Comment:
```suggestion
		sizeLogger := r.log.WithName("volume-size-checker")
```

or something similar but size can be confusing for ops guys

Reply:
`WithName` appends to the already given name which is `machine-reconciler`. I think `machine-reconciler-volume-size-checker` is too verbose.

Reply:
ok, i agree but size is still confusing because it can be size of vm (cpu, memory). Maybe volume-size?